src/includes/performance/traces-sampler-as-sampler/java.spring.mdx

```diff
@@ -36,24 +36,29 @@ import javax.servlet.http.HttpServletRequest
 @Component
 class CustomTracesSamplerCallback : TracesSamplerCallback {
   override fun sample(context: SamplingContext): Double {
-    val request = context.customSamplingContext!!["request"] as HttpServletRequest
-    return when (request.requestURI) {
-      "/payment" -> {
-        // These are important - take a big sample
-        0.5
-      }
-      "/search" -> {
-        // Search is less important and happen much more frequently - only take 1%
-        0.01
-      }
-      "/health" -> {
-        // The health check endpoint is just noise - drop all transactions
-        0.0
-      }
-      else -> {
-        // Default sample rate
-        0.1
+    val customSamplingContext = samplingContext.customSamplingContext
```

Comment:
Good catch!

We should be cautious about code snippets that can be copied in the wrong context and break customers.